client/views/admin/invites/InviteRow.tsx

```diff
@@ -29,7 +47,13 @@ function InviteRow({ _id, createdAt, expires, days, uses, maxUses, onRemove }) {
 		return t('Never');
 	};
 
-	const maxUsesLeft = ({ maxUses, uses }) => {
+	const maxUsesLeft = ({
+		maxUses,
+		uses,
+	}: {
+		maxUses: IInvite['maxUses'];
+		uses: IInvite['uses'];
+	}): number | string => {
 		if (maxUses > 0) {
 			if (uses >= maxUses) {
```

Comment:
Here we can decouple a method for excededMaximun or something like this I think

Reply:
I don't think it deserves a method to check this simple thing!

Reply:
When someone goes offer manutention on this code, maybe will be better if can read what are you checking directly and not to be necessary to interpret the two-block of validation, that was my motivation to suggest this for you, but I'll approve the review.